CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ add_custom_target(rclcs ALL
 else()
 add_custom_target(rclcs ALL
 	COMMAND 
-	"msbuild" rclcs.csproj /property:OutputPath=${CMAKE_BINARY_DIR} /property:AssemblyName=rclcs /property:AllowUnsafeBlocks=true /property:Platform=x64 /property:GenerateDocumentation=true /property:DocumentationFile="rclcs.xml"
+	"msbuild" rclcs.csproj /property:OutputPath=${CMAKE_BINARY_DIR} /property:AssemblyName=rclcs /property:Configuration=Release /property:Platform=x64 /property:AllowUnsafeBlocks=true /property:DocumentationFile="rclcs.xml"
```

Comment:
I revoked that diff, however it was somewhat necessary for me in order to sync the rest of the libs. But maybe also not :)